eng/clipack/Common.projitems

```diff
@@ -92,7 +92,7 @@
       <_MaxFileSizeMB Condition="'$(_MaxFileSizeMB)' == '' and !$([System.OperatingSystem]::IsWindows())">25</_MaxFileSizeMB>
 
       <!-- pwsh on windows. Use Ceiling to match `du -m` behavior -->
-      <_GetFileSizeMBCommand Condition="$([System.OperatingSystem]::IsWindows())">pwsh -Command &quot;[Math]::Ceiling((Get-Item '$(_NativeBinaryPath)' -ErrorAction Stop).Length / 1MB)&quot;</_GetFileSizeMBCommand>
+      <_GetFileSizeMBCommand Condition="$([System.OperatingSystem]::IsWindows())">pwsh -NoProfile -Command &quot;[Math]::Ceiling((Get-Item '$(_NativeBinaryPath)' -ErrorAction Stop).Length / 1MB)&quot;</_GetFileSizeMBCommand>
```

Comment:
This will come in https://github.com/dotnet/aspire/pull/12280. I made it here as well to fix the local build.